editor/plugins/tile_set_editor_plugin.cpp

```diff
@@ -599,7 +599,9 @@ void TileSetEditor::_on_textures_added(const PoolStringArray &p_paths) {
 	}
 }
 
-void TileSetEditor::_on_edit_mode_changed(int p_edit_mode) {
+void TileSetEditor::_on_edit_mode_changed(int p_edit_mode) {	
```

Comment:
You added a trailing tab here, `clang-format` will complain. See http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html